mycroft/util/lang/parse_nl.py

```diff
@@ -0,0 +1,1480 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright 2019 Mycroft AI Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+from collections import namedtuple
+from datetime import datetime, timedelta
+
+from dateutil.relativedelta import relativedelta
+
+from mycroft.util.lang.parse_common import is_numeric, look_for_fractions
+from mycroft.util.lang.common_data_nl import _ARTICLES, _NUM_STRING_NL, \
+    _LONG_ORDINAL_STRING_NL, _LONG_SCALE_NL, \
+    _SHORT_SCALE_NL, _SHORT_ORDINAL_STRING_NL
+
+import re
+
+
+def _invert_dict(original):
+    """
+    Produce a dictionary with the keys and values
+    inverted, relative to the dict passed in.
+
+    Args:
+        original dict: The dict like object to invert
+
+    Returns:
+        dict
+
+    """
+    return {value: key for key, value in original.items()}
+
+
+# negate next number (-2 = 0 - 2)
+_NEGATIVES = {"min", "minus"}
+
+# sum the next number (twenty two = 20 + 2)
+_SUMS = {'twintig', '20', 'dertig', '30', 'veertig', '40', 'vijftig', '50',
+         'zestig', '60', 'zeventig', '70', 'techtig', '80', 'negentig', '90'}
+
+_MULTIPLIES_LONG_SCALE_NL = set(_LONG_SCALE_NL.values())
+
+_MULTIPLIES_SHORT_SCALE_NL = set(_SHORT_SCALE_NL.values())
+
+# split sentence parse separately and sum ( 2 and a half = 2 + 0.5 )
+_FRACTION_MARKER = {"en"}
+
+# decimal marker ( 1 point 5 = 1 + 0.5)
+_DECIMAL_MARKER = {"komma", "punt"}
+
+_STRING_NUM_NL = _invert_dict(_NUM_STRING_NL)
+_STRING_NUM_NL.update({
+    "half": 0.5,
+    "driekwart": 0.75,
+    "anderhalf": 1.5,
+    "couple": 2
```

Comment:
Couple is still english. I think this best translate to "dubbel"

Reply:
I opted for "Paar" as the english parser was tailored to recognize "Couple of hours" which translates to the Dutch "Paar uur"